### DISCUSSION TITLE
What exactly is the process to install comfyui on my pc

### DISCUSSION THREAD

**Discussion by Dralokonda** [lllyasviel/Fooocus#3691] [Q&A]

**Comment by IPv6** [comment]
Why are you asking about Comfyui install in Fooocus issues?
This is ridiculous

> no instructions on installation for cpu

This is because all this stuff practically unusable without dedicated card. You can try and cry, but better rent a server with preinstalled stuff to learn what you need, first. Sad, but there are no simple ways for non-GPU users

**Reply by Dralokonda** [reply]
sorry, my mistake
